apps/api/src/services/course-clone.ts

```diff
@@ -0,0 +1,380 @@
+import { getSupabase } from '$src/utils/supabase';
```

Comment:
@Chifez 

This file should be in `/services/course/clone.ts`. You should put the `clone.ts` file into a course folder.